src/components/SolvroLogo.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from "next/image";
+
+export const SolvroLogo = () => {
+  return (
+    <a href="https://planer.solvro.pl/" className="inline-block">
+      <Image
+        src="/assets/logo/logo_solvro_mono.png"
+        alt="Logo Koła Naukowego Solvro"
+        width={50}
+        height={50}
+        className="cursor-pointer"
```

Comment:
cursor-pointer raczej na linku powinien być?

Reply:
nie tykalem tego i wyrzucilem do osobnego pliku

Reply:
wieem, ale jak możesz to popraw i wrzuć to na Link

Reply:
g